src/main/java/org/isf/security/LogoutApi.java

```diff
@@ -21,40 +21,29 @@
  */
 package org.isf.security;
 
-import javax.validation.Valid;
-
 import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
-import io.swagger.annotations.ApiResponse;
-import io.swagger.annotations.ApiResponses;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
 
 /**
  * Here only for swagger
  *
  * @author antonio
  */
-@Api("Login")
+
 @RestController
-public class LoginApi {
-    /**
-     * Implemented by Spring Security
-     */
-    //@ApiOperation(value = "Login", notes = "Login with the given credentials.")
-    //@ApiResponses({@ApiResponse(code = 200, message = "", response = LoginResponse.class)})
-    //@PostMapping(value = "/auth/login")
-    void login(@Valid @RequestParam String username,  @RequestParam String password) {
-    	 throw new IllegalStateException("Add Spring Security to handle authentication");
-    }
+@Tag(name = "Logout")
```

Comment:
```suggestion
@Tag(name = "Login")
```

So in Swagger the two endpoints are listed under the same tag:

![image](https://github.com/informatici/openhospital-api/assets/2938553/298a45db-b415-4d92-8c86-0ffac7acc72d)

instead of

![image](https://github.com/informatici/openhospital-api/assets/2938553/978d3894-0591-4c1e-aa55-1fdd2dbd5447)

Reply:
Also please review (and accept if ok) my PR https://github.com/intesys/openhospital-api/pull/77